core/query/annotated.go

```diff
@@ -34,7 +34,7 @@ type AnnotatedInput struct {
 	Amount          uint64             `json:"amount"`
 	IssuanceProgram chainjson.HexBytes `json:"issuance_program,omitempty"`
 	ControlProgram  chainjson.HexBytes `json:"control_program,omitempty"`
-	SpentOutput     *SpentOutput       `json:"spent_output,omitempty"`
```

Comment:
I'm not a big fan of the existing `spent_output` attribute because it's an exception against the flat objects we have so far. It's the only place where we use a nested object.